ci-operator/step-registry/storage/conf/storageclass/pre-set-default-gcp/storage-conf-storageclass-pre-set-default-gcp-commands.sh

```
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat << EOF > ${SHARED_DIR}/manifest_storageclass.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)